coreUtilities/src/main/java/org/opensearch/migrations/transform/ListKeyAdaptingCaseInsensitiveHeadersMap.java

```diff
@@ -18,7 +18,7 @@
  */
 @EqualsAndHashCode(callSuper = false)
 public class ListKeyAdaptingCaseInsensitiveHeadersMap extends AbstractMap<String, Object> {
-    protected final StrictCaseInsensitiveHttpHeadersMap strictHeadersMap;
+    public final StrictCaseInsensitiveHttpHeadersMap strictHeadersMap;
```

Comment:
Why did you have to open up access?

Reply:
and it should stay in the replayer.  It only exists as a more efficient implementation of headers maps for our own purposes.